resources/python/build_for_ultimaker.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+#
+# Copyright (C) 2019 Ultimaker B.V.
+#
```

Comment:
Shall we make a variant of this that can run on MacOS and Windows, for those developers that have to work on those platforms? It's intended for the developers to be able to run locally, after all.